js/libs/ui-shared/src/utils/darkMode.ts

```diff
@@ -1,19 +0,0 @@
-const DARK_MODE_CLASS = "pf-v5-theme-dark";
-const mediaQuery = window.matchMedia("(prefers-color-scheme: dark)");
-
-function updateDarkMode(isEnabled: boolean) {
-  const { classList } = document.documentElement;
-
-  if (isEnabled) {
-    classList.add(DARK_MODE_CLASS);
-  } else {
-    classList.remove(DARK_MODE_CLASS);
-  }
-}
-
-export function initializeDarkMode() {
-  updateDarkMode(mediaQuery.matches);
-  mediaQuery.addEventListener("change", (event) =>
-    updateDarkMode(event.matches),
-  );
-}
```

Comment:
No more dark theme for account and admin? Or you should add this to the index.ftl

Reply:
Ahh, I forgot to push some local changes, should be good now.